SOURCES/MAKEFILE

```diff
@@ -50,8 +49,22 @@ OBJETS7  = msg_cust.obj
 
 EXE      = LBA0
 
-$(EXE).exe: $(OBJETS1) $(OBJETS2) $(OBJETS3) $(OBJETS4) $(OBJETS5) $(OBJETS6) $(OBJETS7)
-	echo "run LINK.bat under 4DOS"
+$(EXE).exe: LINK.LD
+	$(WL) Name $@ @ $<
+
+LINK.LD: $(OBJETS1) $(OBJETS2) $(OBJETS3) $(OBJETS4) $(OBJETS5) $(OBJETS6) $(OBJETS7)
+	if exist $@ del $@
+	echo option stack=7000 >> $@
+	echo system dos4g >> $@
+	for %%G in ($<) do echo File %%G >> $@
+	echo Library %LIB386_PATH%\LIB_3D\LIB_3D.LIB >> $@
```

Comment:
This will output absolute paths in the link.ld file. Can we make them relative? Or maybe add link.ld to .gitignore?

Reply:
I can do even better than that, I can remove the need for the file altogether :D